src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,6 @@
-const getEmployerRole = (employeeName, employees) => {};
-
-module.exports = getEmployerRole;
+const getEmployerRole = (employeeName, employees) => {
+    const getEmployee = employees.find( (employee) => employeeName === employee.name);
```

Comment:
Very nice solution, and you could even go a step forward, by instead of assigning the result of the find method to the getEmployee variable and then access the role on it with dot notation, you could just add the .role directly at the end of the find method after the brackets: ... === employee.name).role
and then you can loose the const getEmployee and the curly braces :)